test/product-search-test.js

```diff
@@ -59,6 +59,98 @@ describe('Product Search', function() {
 
   });
 
+  it('summary should return appropriate products from store/s or brand/s 1', function(done){
```

Comment:
@paambaati They are checking the same case with different inputs. Won't description help to identify that the same case is tested with 2 different inputs?